### PR TITLE
Imp/new color pallete

### PR DESCRIPTION
**Pull request by rlebre**
learning pack style like main dicoogle website

### CODE REVIEW

**Comment by Enet4** [comment]
Can you update the [README.md](https://github.com/rlebre/dicoogle-learning-pack/blob/91b25df2a4406cbef71059e78c8f34fa7669a4e8/README.md) file to use the new logo as well?

File: _sass/_style.scss [hunk 1]
```diff
@@ -35,35 +54,60 @@
...
 
 #footer_wrap {
-  background: #214851;
+  background: #383838;
+  color: white;
+  // padding: 2.5rem 0;
+}
+
+#footer_wrap footer p{
+  font-size: 10px;
```
Comment (Enet4) [inline _sass/_style.scss]: The footer text looks too small on my screen. How what do you think of using 12px instead?

Comment (rlebre) [reply _sass/_style.scss]: it can be, but on dicoogle-website is too much

Comment (Enet4) [reply _sass/_style.scss]: I see, but right now the text fits in just fine here. We can change that later in the event that we make significant changes to the footer.

**Pull request merged**